MonoXAnalysis/cfg/run_monojet_cfg.py

```diff
@@ -22,14 +22,15 @@
 doT1METCorr = getHeppyOption("doT1METCorr",True)
```

Comment:
why do you need to modify run_monojet_cfg.py if you added the other cfg?
I would leave this as it is to avoid conflicts